README.md

```diff
@@ -25,12 +25,11 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 
 ## Status and released versions
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
-* **The latest available release and version of Quality-On-Demand (QoD) API is 0.8.1. This is the second alpha version of the API.** There are bug fixes to be expected and incompatible changes in upcoming releases. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
-* Version 0.8.1 of the API is available within the [release-0.8.1 branch](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1):
-  * API [definition](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1/code/API_definitions)
-  * API [documentation](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1/documentation/API_documentation)
+* **The latest available release and version of Quality-On-Demand (QoD) API is 0.9.0. This is the third alpha version of the API.** There are bug fixes to be expected and incompatible changes in upcoming releases. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+* Release 0.9.0 of the API is available within the [release-0.9.0 branch](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0):
+  * [API definition with inline documentation](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0/code/API_definitions)
   * For changes see [CHANGELOG.md](https://github.com/camaraproject/QualityOnDemand/blob/main/CHANGELOG.md)
-* The previous version v0.8.0 of the QoD API is available within the [release-0.8.0 branch](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0)
+* The previous version v0.8.1 of the QoD API is available within the [release-0.8.0 branch](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1)
```

Comment:
Link text should have been updated to [release-0.8.1 branch]

Reply:
That's one of the reasons I always ask for a review :-) ... done in https://github.com/camaraproject/QualityOnDemand/pull/193/commits/4397750dd1f9ca1fee053098df2b1e21d53f591c